R/dplyr-dm.R

```diff
@@ -0,0 +1,65 @@
+#' Rename one or more columns of a [`dm`] table
+#'
+#' Rename columns of your [`dm`] with a similar syntax to `dplyr::rename()`.
+#'
+#' @inheritParams cdm_filter
+#' @param ... One or more unquoted expressions separated by commas. You can treat
```

Comment:
Instead of copy-pasting, would it work to use `@inheritParams dplyr::select` ?

Reply:
I slightly changed the text here and there, 2 reasons:
1. We don't have `cdm_select()` yet
2. Some things would not be quite right

don't have a strong opinion though